### PR TITLE
revive original spreadsheet endpoint

### PR DESCRIPTION
**Pull request by amnonkhen**
ticket: ebi-ait/dcp-ingest-central#403
related to ebi-ait/dcp-ingest-central#13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,10 +32,16 @@ Set `FLASK_APP` beforehand:
 
 ```bash
 export FLASK_APP=broker_app
+```
+
+then start flask
```
Comment (jacobwindsor) [inline README.md:37]: I'm confused. Didn't we just run `python broker_app.py` before? Isn't that easier?

File: broker/service/spreadsheet_storage/spreadsheet_storage_exceptions.py [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
     pass
 
+
 class SubmissionSpreadsheetDoesntExist(Exception):
-    pass
\ No newline at end of file
+    def __init__(self, submission_uuid: str, missing_path: str):
+        self.submission_uuid = submission_uuid
+        self.missing_path = missing_path
+    pass
```
Comment (jacobwindsor) [inline broker/service/spreadsheet_storage/spreadsheet_storage_exceptions.py:10]: Why not make a nice exception message here?

File: broker_app.py [hunk 3]
```diff
@@ -34,25 +31,23 @@
...
-SPREADSHEET_STORAGE_DIR = os.environ.get('SPREADSHEET_STORAGE_DIR')
-
-SPREADSHEET_UPLOAD_MESSAGE = "We’ve got your spreadsheet, and we’re currently importing and validating the data. \
-Nothing else for you to do - check back later."
-
-SPREADSHEET_UPLOAD_MESSAGE_ERROR = "We experienced a problem while uploading your spreadsheet"
 
 def create_app():
     app = Flask(__name__, static_folder='static')
+    app.SPREADSHEET_STORAGE_DIR = os.environ.get('SPREADSHEET_STORAGE_DIR')
```
Comment (jacobwindsor) [inline broker_app.py:37]: Muuuuch better! Thanks

File: README.md (hunk 1 above)
Comment (amnonkhen) [reply README.md:37]: no. I think flask cli does a lot of magic stuff that saves time and effort.

File: broker_app.py (hunk 3 above)
Comment (amnonkhen) [reply broker_app.py:37]: My pleasure :)

File: README.md (hunk 1 above)
Comment (amnonkhen) [reply README.md:37]: With flask run we can use .flaskenv, select the port and host externally and much more.

File: broker/service/spreadsheet_storage/spreadsheet_storage_exceptions.py (hunk 2 above)
Comment (amnonkhen) [reply broker/service/spreadsheet_storage/spreadsheet_storage_exceptions.py:10]: Do you mean setting the text of the message in the exception class?

Comment (jacobwindsor) [reply broker/service/spreadsheet_storage/spreadsheet_storage_exceptions.py:10]: Yes

**Review by jacobwindsor — DISMISSED** [review]

**Review by jacobwindsor — APPROVED** [review]

**Review by yusra-haider — APPROVED** [review]

**Pull request merged**